### ISSUE TITLE
Upgrade ember-basic-dropdown

### ISSUE THREAD

**Issue by chbonser** [miguelcobain/ember-paper#1098]
`ember-basic-dropdown` made major changes in the 2.x release which is not currently compatible with ember-paper.

**Comment by xomaczar** [comment]
And `ember-power-select` has been upgraded to v3.x and is incompatible with ember<3.11. Ember-paper is still in 1.beta cycle and tests all the way back to ember@2.18.
Are there any plans to cut 1.0 version, and start on 2.x which only supports Ember>=3.11

**Comment by LucasHillDex** [comment]
@miguelcobain we love this addon but the above question would be good to get a sense of. Is work on this addon stalled out? Does the community need to pull together to keep it going? What needs to happen to get to 1.0?

**Comment by esbanarango** [comment]
@miguelcobain Let us know if you want/need an extra ✋. 🙏

**Comment by miguelcobain** [comment]
The problem here is that newer versions of ebd/eps require ember 3.11.
I think that semver does not allow us to release ember-paper 1.0.0 with this breaking change: https://jubianchi.github.io/semver-check/#/^1.0.0-beta.26/1.0.0

So, in theory this should require a bump to 2.0.0, is that right?

Any suggestions?

**Comment by miguelcobain** [comment]
So, I rebuilt paper-menu, paper-select and paper-autocomplete using the new ebd and eps versions. It's on master.

All tests are passing, but it would be great if you guys could test these on your apps as well.

This also includes other changes like using `html-next/vertical-collection` instead of virtual-each. Also removes paper-virtual-repeat, encouraging the use of vertical-collection.

@betocantu93 this also includes some changes to these components directories.
Basically every `paper-autocomplete-*` component is now inside a paper-autocomplete directory.

This requires some updates on the blacklist/whitelisting feature that you built.
Could you please update it to work on the new directory structure?

**Comment by alex9nadon** [comment]
[cid:864F6B10-9C1E-4B7F-83B4-21085910C4A5-L0-001]

Alexandre L. Nadon, ing jr.
450-822-8226

On Dec 27, 2019, at 11:57 AM, Miguel Andrade <notifications@github.com> wrote:

﻿

So, I rebuilt paper-menu, paper-select and paper-autocomplete using the new ebd and eps versions. It's on master.

All tests are passing, but it would be great if you guys could test these on your apps as well.

This also includes other changes like using html-next/vertical-collection instead of virtual-each. Also removes paper-virtual-repeat, encouraging the use of vertical-collection.

@betocantu93<https://github.com/betocantu93> this also includes some changes to these components directories.
Basically every paper-autocomplete-* component is now inside a paper-autocomplete directory.

This requires some updates on the blacklist/whitelisting feature that you built.
Could you please update it to work on the new directory structure?

**Comment by betocantu93** [comment]
@miguelcobain great, on it. I think I don't need to worry about styles, right? those come from angular-material-styles and the treeForStyles logic should remain intact?

**Comment by miguelcobain** [comment]
@betocantu93 your work was also selectively importing the needed styles. But yes, everything regarding styles should still apply.

**Comment by betocantu93** [comment]
Here's the PR #1118

**Comment by LucasHillDex** [comment]
@miguelcobain I think a 2.0 release makes sense. A bit funny but, won't hose folks depending on the beta releases.

**Comment by miguelcobain** [comment]
After trying the branch for some time on our app, everything seems to work normally.
Released this in `1.0.0-beta.27`.

**Comment by chbonser** [comment]
@miguelcobain thanks for the release. Did the latest `ember-basic-dropdown` release add broader Ember support or does `1.0.0-beta.27` require > Ember 3.11? If so it might be worth a mention in the change log.

**Comment by miguelcobain** [comment]
@chbonser Yes, indeed ember-paper `1.0.0-beta.27` requires > Ember 3.11. I will add it to the changelog.
